docs/node/migrations/zero-downtime-migrations/tutorial.mdx

```diff
@@ -0,0 +1,158 @@
+---
+sidebar_position: 3
+sidebar_label: "Tutorial"
+id: zero-downtime-migrations-tutorial
+title: Zero Downtime Migrations Tutorial
+description: "Tutorial on how to upgrade a network using zero-downtime migrations"
+slug: /node/migrations/zero-downtime-migrations/tutorial
+---
+
+This tutorial demonstrates how to ugrade a network using zero-downtime migrations. 
+
+## Migration Proposal Consensus
+
+Let's assume that the validators decided off-chain to migrate the network to a new chain `kwil-chain-2` and agreed upon the below migration proposal:
+```
+ActivationPeriod: 100
+Duration: 2000
+ChainID: "kwil-chain-2"
+```
+
+### Submit Migration Proposal
+A validator operator can submit a migration proposal to the network by running the below command. 
+
+```bash
+kwil-admin migrate propose --activation-period 100 --duration 2000 --chain-id kwil-chain-2 --rpcserver /tmp/kwild_0.socket
```

Comment:
We should remove the `--rpcserver ...` part of the command everywhere on this page.

I don't think this tutorial needs to be written as a workable example with multiple nodes on the same machine.  It's less confusing just to say things like "on each node" or "on the new node" etc.